config/settings.yml

```diff
@@ -1480,6 +1480,6 @@
       :memory_threshold: 1.gigabytes
       :nice_delta: 1
     :websocket_worker:
-       :connection_pool_size: 8
+       :connection_pool_size: 26
```

Comment:
@skateman do you need this for euwe?  Can you try @NickLaMuro's PR to see if that helps?  If you need it for euwe, we should do a surgical fix to allow the websocket worker more connections until we can upgrade to a version of rails containing the fix for the "query cache middleware establishing connections in each thread."